modules/doc/content/getting_started/installation/test_moose.md

```diff
@@ -1,9 +1,9 @@
-## Compile and Test MOOSE
+## Build and Test MOOSE
```

Comment:
SHould you use the template you created ?

modules/doc/content/getting_started/installation/test_moose.md.template

Reply:
We can't... If I am thinking correctly when I say we may not even need `test_moose.md.template` anymore (I can't remember. But I know when I tried to use the other templates we fail; clone_moose.md.template and the other one).

I ran into template recursion issues when attempting to use templates for our basic instructions. So these continue to need not use templates (keep the originals). But as far as our Advanced instructions, the templates work.

Sorta makes me wonder if its even worth templating for just two pages.

Reply:
now that I am staring at the pages, yes... we should use templates!

Reply:
What urks me... is that the page renders in the end, just fine.

Reply:
oh well. It's not the biggest re-use, you're removing a lot of those re-uses here anyway

Reply:
@cticenhour  helped me figure out the issue :) I was doing it wrong! Of course.

So now, moar templates incoming!